extensions-api/src/main/scala/pl/touk/nussknacker/engine/modelconfig/ModelConfigLoader.scala

```diff
@@ -100,8 +101,11 @@ abstract class ModelConfigLoader extends Serializable with LazyLogging {
       service1Url: ${baseUrl}/service1
       and have baseUrl taken from application config
      */
-    // We want to respect overrides (like system properties) and standard fallbacks (like reference.conf)
-    ConfigFactory.load(classLoader, withFallbackToDefaults(inputConfig, classLoader))
+    // We want to respect standard fallbacks (like reference.conf), but leave resolution of overrides and environment
```

Comment:
Actually, resolution of environment variables is sometime that here as well (behind `resolve(ConfigResolveOptions.defaults)`. We only skip system properties resolution and environment variables override mechanism

Reply:
Can you give an example? I tried to write a test for that (in `DefaultModelConfigLoaderTest`) and it fails in the call you mentioned with `Could not resolve substitution to a value: ${PATH}`.

Modified defaultModelConfig.conf:
```conf
{
  property1: shouldBeOverridenToo
  property2: ${property1}Suffix
  otherProperty: true
  envPathProperty: ${PATH}
}
```

Test:
```scala
  test("should resolve environment variables") {
    val config = LocalModelData(inputConfig, List.empty).modelConfig
    val envPath = System.getenv("PATH")

    envPath shouldNot be(null)
    config.getString("envPathProperty") shouldBe envPath
  }
```

Reply:
strange, I do the same on your branch and it works for me :) https://github.com/TouK/nussknacker/pull/5421/files